k8s/cmd/README.md

```diff
@@ -2117,3 +2162,6 @@ vineyardctl schedule workload [flags]
       --vineyardd-namespace string   the namespace of vineyard cluster (default "vineyard-system")
 ```
 
+er (default "vineyard-system")
+```
+
```

Comment:
Please check the generated doc manually.

Reply:
Done.